root/static/scripts/edit/MB/edit.js

```diff
@@ -202,8 +202,9 @@ import request from '../../common/utility/request';
                     return event;
                 }
 
-                return null;
-            }).compact().value();
+            })
+            .compact()
+            .value();
```

Comment:
These should be indented further, and the `.map` above should be moved to the next line and indented too.